compiler/next/include/chpl/queries/stringify-functions.h

```diff
@@ -127,17 +128,44 @@ static inline void defaultStringifyMap(std::ostream& streamOut,
     std::sort(mapVec.begin(), mapVec.end());
     for (auto const& x : mapVec)
     {
-      streamOut << separator
-                << keyString(stringKind, x.first)
-                << ": "
-                << valString(stringKind, x.second);
+      streamOut << separator;
+      keyString(streamOut, stringKind, x.first);
+      streamOut << ": ";
+      valString(streamOut, stringKind, x.second);
 
       separator = ", ";
     }
     streamOut << "}";
   }
 }
 
+// stringify a map by stringifying each key, value pair
+// uses {k1: v1, k2: v2} formatting
+  template<typename K, typename V>
+  static inline void defaultStringifyMap(std::ostream& streamOut,
+                                         StringifyKind stringKind,
+                                         const std::map<K,V>& stringMap)
+  {
+    if (stringMap.size() == 0) {
+      streamOut << "{ }";
+    } else {
+      std::string separator;
+      stringify<K> keyString;
+      stringify<V> valString;
+      streamOut << "{";
+      for (auto const& x : stringMap)
+      {
+        streamOut << separator;
+        keyString(streamOut, stringKind, x.first);
+        streamOut << ": ";
+        valString(streamOut, stringKind, x.second);
+
+        separator = ", ";
```

Comment:
I personally like to omit the trailing comma after the last element, but if other don't care I won't.

Reply:
This shouldn't produce a trailing comma, the separator is placed before the element. Here are the examples from the tests for `map` and `unordered_map`
```
{1: "Jan.", 2: "Feb.", 3: "Mar."}
{0: "Sun.", 1: "Mon.", 2: "Tue.", 3: "Wed."}
```

Reply:
Having the separator be empty and then initialized to `", "` is kind of cool, my eyes totally glossed over that pattern.